### ISSUE TITLE
Sounds Getting "muted"

### ISSUE THREAD

**Issue by Dangomation** [QuantumFusionMC/DashLoader-Definitions#35]
**Describe the bug**
When I go to play the Modpack with Dashloader installed the game gets muted within about 2 min.

**To Reproduce**
I'm unsure the best way to reproduce this error. 
I've only just found this out myself an hour ago.
I just have a Modpack and enabling Dashloader mutes the sound very consistently when enabled.

**Expected behavior**
Not to have sounds muted when enabled.

**Context (please complete the following information):**
 - DashLoader Version [4.1.3.]
 - Minecraft Version [1.18.2]

**Full Logs**
https://paste.gg/p/anonymous/64a725e4b7464a0d888b269cc1f6034e

**Additional context**
I hope thats enough to get started. I'd hate not to use dashloader because of a silly sound bug.

**Comment by alphaqu** [comment]
Hey can you please try to add this into your dashloader.json config and see if that resolves your issue.
```diff
{
  "options": {
++  "FAST_SOUND_FILE_SKIP": false
  },
  "compression"
  ...
}
```

**Comment by Dangomation** [comment]
This did seem to be a solution for this bug. I guess managing those sound files can be tricky sometimes.

**Comment by alphaqu** [comment]
Fixed in 5.0 release
